### PR TITLE
Intake overcurrent fixes

### PR DESCRIPTION
**Pull request by maxh65000**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -36,7 +36,8 @@
     READY_TO_INTAKE,
     INTAKE,
     STORE_BALL,
-    STORAGE_COMPLETE
+    STORAGE_COMPLETE,
+    BARF
```
Comment (jhinspeter) [inline src/main/java/frc/robot/Robot.java]: Can we rename this? Perhaps EJECT_BALLS

Comment (jhinspeter) [inline src/main/java/frc/robot/Robot.java]: Can we rename this? Perhaps EJECT_BALLS

**Review by onedeadeye — APPROVED** [review]
cool and good

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -357,6 +356,10 @@ public void RobotLoop() {
       mRobotLogger.log("Changed state to " + mShootingState);
     }
 
+    if(mOperatorInterface.isBarf()){
```
Comment (jhinspeter) [inline src/main/java/frc/robot/Robot.java]: Run a linter please. therearenospacesinyourcode

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -460,15 +460,18 @@ private void executeIntakeStateMachine() {
         
         mIntake.resetOvercurrentCooldown();
         break;
+      case BARF:
```
Comment (jhinspeter) [inline src/main/java/frc/robot/Robot.java]: How do we ever get out of this state? Seems like we should transition to IDLE after a pre-configured amount of time.

**Review by jhinspeter — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (maxh65000) [reply src/main/java/frc/robot/Robot.java]: Done

Comment (maxh65000) [reply src/main/java/frc/robot/Robot.java]: Done

File: src/main/java/frc/robot/Robot.java (hunk 3 above)
Comment (maxh65000) [reply src/main/java/frc/robot/Robot.java]: Done

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (maxh65000) [reply src/main/java/frc/robot/Robot.java]: Fixed

**Review by jhinspeter — APPROVED** [review]

**Pull request merged**